Assets/Script/Bullet/BulletService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletService : SingletonBehaviour<BulletService>
+{
+    public BulletView bulletView;
+    // Start is called before the first frame update
+
+    public void Start()
+    {
+        //CreateBullet();
+    }
+    public void CreateBullet(Vector3 position)
```

Comment:
return

Reply:
Sir , Why that it should be returning bulletController?